packages/extension/src/components/organisms/TokenListing/TokenListing.tsx

```diff
@@ -82,6 +85,13 @@ export const TokenListing: FC<TokenListingProps> = ({ address }) => {
     setExplanationOpen(false);
   }, []);
 
+  const handleFundWallet = useCallback(async () => {
+    setXRPBalance(LOADING_STATE);
+    fundWallet()
+      .then(({ balance }) => setXRPBalance(balance.toString()))
+      .catch((e) => setXRPBalance(ERROR_STATE));
```

Comment:
Could you please send the error within Sentry as well? This will help us to log the errors as well.
```suggestion
      .catch((e) => {
         Sentry.captureException(e);
         setXRPBalance(ERROR_STATE)
      });
```